spec/factories/items.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :item do
+    name "Gift1"
```

Comment:
Not sure why, usually I see sequence being used in generating one specific attribute - for example e-mail address or dynamic name (e.g. in block). Are you sure?